symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py

```diff
@@ -0,0 +1,37 @@
+from sympy import Eq, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The volt-ampere characteristic of a vacuum diode is described by the law of degree three of the second. The diode
+## constant that is in this law depends only on the relative position, shape and size of the electrodes of the vacuum diode.
+
+## Law is: I = g * U^(3 / 2), where
+## I - anode current,
```

Comment:
Is it same as cathode current?

Reply:
> Is it same as cathode current?

No, the anode current and the cathode current are not equal.